src/app/configs/enum-helper.ts

```diff
@@ -0,0 +1,13 @@
+export enum taskStatuses 
+{
+    "To do" = 1,
+    "In progress" = 2,
+    "In review" = 3,
+    "Compleated" = 4
+}
```

Comment:
use created endpoint create the tech dept ticket for this

Reply:
https://app.clickup.com/t/1tmxt4h